app/main.py

```diff
@@ -2,0 +1,26 @@
+import os
+
+
+def move_file(command: str) -> None:
+    instructions = command.split()
+    source = instructions[1]
+    destination = instructions[2]
```

Comment:
Consider using unpacking for the `instructions` list to assign `source` and `destination` directly. This is more efficient and aligns with the checklist recommendation: 'If you need to assign values from a list to individual variables, it's better to use unpacking.'